### ISSUE TITLE
Add forwardRef to defaultStyle

### ISSUE THREAD

**Issue by jpalumickas** [signavio/react-mentions#411]
After upgrading to 4.0.0, when we try to use ref with MentionsInput, to access MentionsInput functions, we this error:

```
Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?
```

**Steps to reproduce:**

1. Add ref to MentionsInput

**Expected behaviour:**

Allow to use ref with MentionsInput to access class functions

**Observed behaviour:**

Error written above

**Workaround:**

Add forwardRef to defaultStyle might help

**Comment by jfschwarz** [comment]
Yes, this would make sense. Glad to accept a PR, otherwise we're going to add this once we find a moment...
